templates/indicators/data_report.html

```diff
@@ -71,8 +71,8 @@ <h4 class="page-title">Indicator Report</h4>
     <div id="map"></div>
 
     {% include "small_site_profile_map.html" %} 
-    {% elif "graph" in request.get_full_path %} 
-    {% include "indicators/indicator_graphs.html" %} 
+    <!-- {% elif "graph" in request.get_full_path %} 
```

Comment:
Django will still access this peace of code. Please use the Django template block comment syntax to comment out the block of code
`{% comment %}...{% endcomment %}` for multi-line
`{# ....  #}` for single-line